0-bootstrap/README.md

```diff
@@ -133,6 +133,7 @@ your current Jenkins manager (controller) environment.
    ```
 
 1. Use the helper script [validate-requirements.sh](../scripts/validate-requirements.sh) to validate your environment:
+   1. The helper script requires `jq` installed. Visit [https://stedolan.github.io/jq/download/](https://stedolan.github.io/jq/download/) for more information on how to install it.
```

Comment:
Can we just validate this in the script itself so the script can surface this message if jq is not installed.

Reply:
Done